DataManagementSystem/private/FTS3Utilities.py

```diff
@@ -326,9 +334,14 @@ def _randomServerPolicy( self, _attempt ):
   @staticmethod
   def _getFTSServerStatus( ftsServer ):
     """ Fetch the status of the FTS server from RSS """
-    pub = RPCClient( 'ResourceStatus/Publisher' )
-    res = pub.getElementStatuses( 'Resource', ftsServer, None, None, None, None )
 
+    res = getFTS3ServerName(ftsServer)
+    if not res['OK']:
+      return res
+    ftsServerName = res['Value']
+
+    pub = RPCClient( 'ResourceStatus/Publisher' )
+    res = pub.getElementStatuses( 'Resource', ftsServerName, 'FTS', None, None, None )
```

Comment:
There's a better way to do this:
```
ResourceStatus().getElementStatus(ftsServerName, "FTS")
```
and then you need to replace the import of `RPCClient` with `from DIRAC.ResourceStatusSystem.Client.ResourceStatus import ResourceStatus`